blog.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8">
+		<link rel="stylesheet" href="styles/main.css">
+		<title>Dog Adoption | Blog</title>
+	</head>
+	<body>
+		<header>
+			<div class="container">
+				<div class="title">
+					<div class="left-side">
+						<img src="img/logo.svg" class="logo-img" alt="">
+						<p class="logo-text">Seattle's premiere dog adoption agency</p>
+					</div>
+					<div class="right-side">
+						<img src="img/cart-icon.svg" class="cart-img" alt="">
+						<span class="cart-price">$1234.56</span>
+						<a href="checkout.html" class="button-top">Checkout</a>
+					</div>
+				</div>
+			</div>
+		</header>
+		<nav class="nav-first">
+			<div class="container">
+				<a href="index.html">Home</a>
+				<span class="after"></span>
```

Comment:
You could assign these pseudo elements to the a tags if you target them right. Just so you don't have to have these extra elements in.